<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SichuanRobot</class>
 <widget class="QWidget" name="SichuanRobot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1469</width>
    <height>912</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_19">
   <property name="spacing">
    <number>50</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>卡机TCP</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>IP地址：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_CardMachineIp"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port端口号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_CardMachinePort"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>连接状态：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_connect">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>实现对卡机的控制：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_C1">
            <property name="text">
             <string>C1时间同步</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_C2">
            <property name="text">
             <string>C2设置黄闪报警</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_C3">
            <property name="text">
             <string>C3入口卡机控制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_C4">
            <property name="text">
             <string>C4出口卡机控制</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButton_C5">
            <property name="text">
             <string>C5出票控制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_reserve">
            <property name="text">
             <string>预留</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>接收到卡机返回数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>读卡器</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>IP地址：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_CardReaderIp"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Port端口号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_CardReaderPort"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>注意：下工位21004， 上工位21003</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="1,1">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>连接状态：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_connect_CardReader">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>给读卡器发指令：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="pushButton_OpenCad">
            <property name="text">
             <string>C1 打开读卡器</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_CloseCard">
            <property name="text">
             <string>C2 关闭读卡器</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_FindCard">
            <property name="text">
             <string>C3 开始寻卡</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="pushButton_CPCbaseInfo">
            <property name="text">
             <string>C4 读CPC 基本信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ReadCPC_EF01">
            <property name="text">
             <string>C5 读CPC EF01信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ReadCPC_EF02">
            <property name="text">
             <string>C6 读CPC EF02信息</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton_ReadCPC_EF04">
            <property name="text">
             <string>C7 读CPC EF04信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_UpdateCPC_EF01">
            <property name="text">
             <string>C8 更新CPC EF01信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_UpdateCPC_EF02">
            <property name="text">
             <string>C9 更新CPC EF02信息</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QPushButton" name="pushButton_UpdateCPC_EF04">
            <property name="text">
             <string>CA 更新CPC EF04信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ReadETCinfo">
            <property name="text">
             <string>CB 读ETC信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_UpdateETCinfo">
            <property name="text">
             <string>CC 更新ETC信息</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QPushButton" name="pushButton_GetTAC">
            <property name="text">
             <string>CD 重取TAC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_GetdAuthorizInfo">
            <property name="text">
             <string>CE 取授权信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ExecOnlineAuthoriz">
            <property name="text">
             <string>CF 执行在线授权</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>接收到读卡器返回数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_CardReader"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
